docs/simple/catalog.jsonnet

```diff
@@ -46,13 +52,12 @@
           'megan.mcdonald',
           'milly',
           'rob',
-          'rory',
         ],
         auxiliary_members: [
           'pete',
-          'lawrence',
           'ben',
         ],
+        escalation_path: $.escalationPaths.RSP_PINC_SP,
```

Comment:
I think maybe we should just reference by ID directly without the ID array - everything else in here feels directly referenced maybe

Reply:
This is how we manage it ourselves, and is more easily updated - I think it's good practice if you assign escalation paths to multiple teams!